### ISSUE TITLE
App crashing if Usage is set to 0

### ISSUE THREAD

**Issue by Bollafant** [njic/medassist#8]
My medication changes from time to time.  So I tried to insert one medicament with an amount but set usage to 0. I also tried this on the Demo-Server, and it also crashed, sorry.
I know this is not a common use case.

Thanks for this app. I hope it could replace my Calc-Sheet. :)

**Comment by njic** [comment]
Thanks for feedback and opening the issue. I'm improving inputs (next release) for adding medication, the plan was to set minimum usage to 0.1, but now I see it could be useful to make 0 usage possible as well. I'm not sure what is the reason for crash at the moment, but I'm sure it will work in the next release

**Comment by njic** [comment]
Hey mate. Please try out new release

https://github.com/njic/medassist/releases/tag/v0.15.0.develop

I fixed multiple issues here. If it turns  out it is stable enough, I'll merge it to main branch and create latest release.

I also moved database path (to achieve uniform path no matter what installation method was chosen), so make sure to update docker-compose

```
    volumes:
      - /path/to/database/directory:/app/medassist
```
      
 Change to:
 
```
     volumes:
      - /path/to/database/directory:/app/database
```
 
 Also change version tag to `develop` or `v0.15.0.develop`
 
 To backup your medication, just save `medication.db` somewhere
 
 
 Please confirm is it working for you so I can close the issue

**Comment by njic** [comment]
"latest" tag has been released together with v0.16.2. I'm closing the issue since I believe its fixed, but feel free to reopen it or let me know if you face the same problem again. I'm fairly confident, but I'm looking forward to your feedback :D

**Comment by Bollafant** [comment]
I tested it today and it works.
Thank you very much for the great app.
